infrastructure/task/scripts/provision-cert-manager.sh

```diff
@@ -78,7 +80,23 @@ isDiffing || kubectl wait \
 isDiffing || echo " > Applying staging and production issuers"
 isDiffing && echo " > Diffing issuers"
 
+# shellcheck disable=SC2016
```

Comment:
Consider adding a comment why we need to disable a shell check

The checks are there for a reason. If we think it is necessary to disable them then adding a comment explaining why is helpful. We also do this a bit higher up in this file.

Reply:
Yeah - will do after the fact. This was just copied from provisioning of nginx and I didn't want the linter to stop me right now